.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: clojure
 lein: lein2
 jdk:
-- oraclejdk7
-- openjdk7
+  - oraclejdk7
+  - openjdk7
+script: ./ext/travisci/test.sh
+notifications:
+  email: false
```

Comment:
Should we go ahead and do the `sudo: false` so we can run on Travis' container-based infrastructure?